tests/common/plugins/sanity_check/recover.py

```diff
@@ -26,13 +27,58 @@ def reboot_dut(dut, localhost, cmd):
         assert False, "Failed to reboot the DUT"
 
     localhost.wait_for(host=dut.hostname, port=22, state="started", delay=10, timeout=300)
-    wait(30, msg="Wait 30 seconds for system to be stable.")
+    wait(120, msg="Wait 120 seconds for system to be stable.")
```

Comment:
Can you make this a constant (e.g. `REBOOT_RECOVERY_WAIT_PERIOD`)?

Reply:
I further refactored it. I have one more idea, to actually put the function into the method dictionary. But i think I should leave that for another PR.